crates/macros/src/config_enum/variant.rs

```diff
@@ -80,7 +80,7 @@ impl Variant<'_> {
 
         if self.args.fallback {
             quote! {
-                Self::#name(fallback) => fallback,
+                Self::#name(fallback) => return std::write!(f, "{fallback}"),
```

Comment:
Does this require `std::fmt::Write` to be in scope?

Reply:
It does, good call. I'll update this to use `write_fmt` instead.